### PR TITLE
feat(payment): PAYPAL-876 added onError handler

### PR DESCRIPTION
**Pull request by andriiVitvitskyi1990**
## What?
PAYPAL-876 added onError handler

## Why?
According to task PAYPAL-876. Added onError handler to display error popup when polling timer expired

## Testing / Proof
tested on dev

@bigcommerce/checkout @davidchin

### CODE REVIEW

**Review by BC-ACherednichenko — APPROVED** [review]
LGTM, 

only one concern that we need to check before the merge

`onError` should be an optional param in `checkout-sdk`,  so we could keep compatibility with earlier checkout versions or custom checkout implementations

File: src/app/payment/paymentMethod/PaypalCommercePaymentMethod.tsx [hunk 1]
```diff
@@ -29,6 +29,9 @@ const PaypalCommercePaymentMethod: FunctionComponent<PaypalCommercePaymentMethod
                 setSubmitted(true);
                 submitForm();
             },
+            onError: (error: any) => {
```
Comment (davidchin) [inline src/app/payment/paymentMethod/PaypalCommercePaymentMethod.tsx:32]: You don't need `any` as the parameter should already be typed.

**Review by davidchin — APPROVED** [review]

File: src/app/payment/paymentMethod/PaypalCommercePaymentMethod.tsx (hunk 1 above)
Comment (andriiVitvitskyi1990) [reply src/app/payment/paymentMethod/PaypalCommercePaymentMethod.tsx]: fixed

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
   },
   "homepage": "https://github.com/bigcommerce/checkout-js#readme",
   "dependencies": {
-    "@bigcommerce/checkout-sdk": "^1.124.1",
+    "@bigcommerce/checkout-sdk": "^1.125.0",
```
Comment (BC-ACherednichenko) [inline package.json:29]: Can you update `package-lock.json` as well?

Comment (andriiVitvitskyi1990) [reply package.json:29]: fixed

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/checkout",
-  "version": "1.110.3",
+  "version": "1.110.0",
```
Comment (BC-ACherednichenko) [inline package-lock.json:3]: It should not change the version on the module, it is managed by circle-ci

Comment (andriiVitvitskyi1990) [reply package-lock.json]: fixed

**Pull request merged**